llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -406,19 +406,19 @@ def FeatureStdExtZcf
 
 def FeatureStdExtZcmp
     : RISCVExtension<"zcmp", 1, 0,
-                     "'Zcmp' (sequenced instuctions for code-size reduction)",
+                     "'Zcmp' (sequenced instructions for code-size reduction)",
```

Comment:
This text appears in a test file generated from this. I'll fix it and make a commit.